network/p2p/network.go

```diff
@@ -109,18 +109,18 @@ func (n *Network) Done() <-chan struct{} {
 // Register will register the given engine with the given unique engine engineID,
 // returning a conduit to directly submit messages to the message bus of the
 // engine.
-func (n *Network) Register(channelID string, engine network.Engine) (network.Conduit, error) {
-	if _, ok := channels.RolesByChannelID(channelID); !ok {
-		return nil, fmt.Errorf("unknown channel id: %s, should be registered in topic map", channelID)
+func (n *Network) Register(channel network.Channel, engine network.Engine) (network.Conduit, error) {
+	if _, ok := channels.RolesByChannel(channel); !ok {
+		return nil, fmt.Errorf("unknown channel: %s, should be registered in topic map", channel)
 	}
```

Comment:
We could add a utility function for this for readability
```suggestion
	if !channels.Exists(channel) {	
		return nil, fmt.Errorf("unknown channel: %s", channel)
	}
```

Reply:
added here https://github.com/onflow/flow-go/pull/289/commits/8b3a21ad501d166b80e44900a018347820e93942